autospec/infile_update_spec.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/true
+#
+# infile_parser.py - part of autospec
+# Copyright (C) 2017 Intel Corporation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import re
+
+
+def update_summary(bb_dict, specfile):
+    """
+    Updates the default summary to the summary or description scraped from
+    the bitbake file IF the default summary is "No detailed summary available"
+    """
+    k = "default_sum"
+    v = "No detailed summary available"
+
+    if hasattr(specfile, k) and getattr(specfile, k) == v:
+        for i in ['SUMMARY', 'DESCRIPTION']:
+            if i in bb_dict:
+                setattr(specfile, k, bb_dict.get(i))
```

Comment:
why not just `specfile.default_sum = bb_dict.get(i)`? Have to be careful though because `.get()` can return `None` and you should default `default_sum` to `""`

Reply:
Do we want to keep default_sum to the "No detailed summary available"?

Reply:
Done